diem-move/diem-framework/src/lib.rs

```diff
@@ -71,7 +72,7 @@ static DPN_FRAMEWORK_PKG: Lazy<CompiledPackage> = Lazy::new(|| {
         test_mode: false,
         generate_docs: true,
         generate_abis: true,
-        install_dir: None,
+        install_dir: Some(tempdir().unwrap().path().to_path_buf()),
```

Comment:
What was happening before when this was None?

Reply:
If no `install_dir` is provided then the build artifact is saved under `<current_dir>/build`.

Reply:
ah. And is this DPN lazy static thing just for tests? is this tempdir thing good enough generally? (won't it get wiped away in a lot of cases?)

Reply:
Yep! this should just be for tests. What I really want is for it to load up the current release, but IIRC we don't want this because we want e.g., the move-lang tests to use a fresh build and not the current saved build.

Reply:
I think this is ready to get in: I ran all the `language` tests about 50-60 times with `cargo nextest` and didn't observe failures due to races coming from the diem framework compilation. I could observe it without these changes every ~10 or so runs.